flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/table/JdbcDynamicTableSourceITCase.java

```diff
@@ -356,6 +356,152 @@ void testLookupJoin(Caching caching) {
         }
     }
 
+    @ParameterizedTest
+    @EnumSource(Caching.class)
+    void testLookupJoinWithFilter(Caching caching) {
+        // Create JDBC lookup table
+        List<String> cachingOptions = Collections.emptyList();
+        if (caching.equals(Caching.ENABLE_CACHE)) {
```

Comment:
nit: 
`caching` is enum i.e. it could be compared with just `=` ... 
also at line 423 it is compared with `=`, it's better to follow same approach across the code